src/lib/OLED/XBMStrings.h

```diff
@@ -131,7 +131,7 @@
 
 #endif
 
-#if defined(USE_OLED_SPI) || defined(USE_OLED_I2C)
+#if defined(USE_OLED_SPI) || defined(USE_OLED_I2C) 
```

Comment:
You should also revert the whitespace change so there are no changes to the file.